File: Assets/GraphQL/Model/Transportation/TransportationRent.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class TransportationRent {
		[GqlSelection("transportationRentID")] public int TransportationRentID { get; private set; }
		[GqlSelection("countryID")] public int CountryID { get; private set; }
		[GqlSelection("transportationTypeID")] public int TransportationTypeID { get; private set; }
		[GqlSelection("companyName")] public string CompanyName { get; private set; }
		[GqlSelection("price")] public float Price { get; private set; }
	}
}

File: Server/src/schema/Transportation/transportationRent.ts
import { Field, Float, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity()
@ObjectType()
export default class TransportationRent extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    transportationRentID!: number;

    @Field(() => Int!)
    @Column()
    countryID!: number;

    @Field(() => Int!)
    @Column()
    transportationTypeID!: number;

    @Field(() => String!)
    @Column()
    companyName!: String;

    @Field(() => Float)
    @Column()
    price!: number;

    constructor() {
        super();

        this.transportationRentID = 0;
        this.countryID = 0;
        this.transportationTypeID = 0;
        this.companyName = "None";
    }
}